package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "@leapfrogtechnology/async-store",
+  "version": "1.0.0-alpha.1",
+  "description": "Async Store",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
```

Comment:
Add files key so that only `dist` folder is published to npm.

```json
"files": [
  "dist"
]
```

Can use `yarn pack` to check what the final tar/zip will have.

Reply:
Nice catch.